Bottom: 67c5218373fbee9b733d3ce4ee79afc6e06d50c5
Top:    9666cc9cf72b2edd2aab7c5d45ba2d2a3c39c87b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-25 17:30:06 +0100

Refresh of draw-c-split-gathering-routine

---

diff --git a/src/draw.c b/src/draw.c
index 05b2272..3e052ad 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -688,7 +688,7 @@ clearPad_callback (const BoxType * b, void *cl)
   PadTypePtr pad = (PadTypePtr) b;
   int *side = cl;
   if (ON_SIDE (pad, *side) && pad->Mask)
-    _draw_pad (Output.pmGC, Pad, true, mask);
+    _draw_pad (Output.pmGC, pad, true, true);
   return 1;
 }
